devices/iluminize/led_controller_multi_5.cct.json

```diff
@@ -0,0 +1,186 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "iluminize",
+  "modelid": "HK-ZD-CCT-A",
+  "product": "LED Controller Multi 5 (CCT mode)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_COLOR_TEMPERATURE_LIGHT",
+      "restapi": "/lights",
+      "uuid": [
+        "$address.ext",
+        "0x01"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "cap/color/capabilities",
+          "refresh.interval": 86400
+        },
+        {
+          "name": "cap/color/ct/max",
+          "refresh.interval": 86400,
+          "default": "0xfeff"
+        },
+        {
+          "name": "cap/color/ct/min",
+          "refresh.interval": 86400,
+          "default": 0
+        },
+        {
+          "name": "cap/color/xy/blue_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/blue_y",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/green_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/green_y",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/red_x",
+          "static": 0
+        },
+        {
+          "name": "cap/color/xy/red_y",
+          "static": 0
+        },
+        {
+          "name": "config/bri/couple_ct",
```

Comment:
Does the controller support this?  I’d default it to false.  Also, I would expect it to support `execute_if_off` as well.

Reply:
If you could tell me the cluster + attribute address I will re-check. 

BTW: I find it very hard with the DDF editor to match against the controller. My workflow is always drop everything in, then compare endpoints if active. It takes too long and easily causes to miss something. Is there a better way to match the written path vs available clusters/addresses?

Reply:
I double checked, but it is as displayed in the screenshots:

Level Control Options bitmap is: `0x01`
Color Control Options bitmap is: `0x01`

do I need to adjust something?